lte/gateway/c/oai/tasks/sgw_s8/sgw_s8_handlers.c

```diff
@@ -1108,3 +1109,19 @@ void sgw_s8_handle_release_access_bearers_request(
   }
   OAILOG_FUNC_OUT(LOG_SGW_S8);
 }
+
+/* Generates random control plane teid and is used for both s11 and
+ * s8 interface to uniquely identify sgw_s8_context for pdn session
+ */
+static teid_t sgw_s8_generate_new_cp_teid(void) {
```

Comment:
if this is generic id generation service, then let's make name generic e.g., `sgw_generate_new_cp_teid`.

Reply:
This function generates control plane teid for sgw_s8 task only. So kept the name as sgw_s8_generate_new_cp_teid